scapy/contrib/mqtt.py

```diff
@@ -1,16 +1,19 @@
-## This file is part of Scapy
-## See http://www.secdev.org/projects/scapy for more informations
-## Copyright (C) Santiago Hernandez Ramos <shramos@protonmail.com>
-## This program is published under GPLv2 license
+''' This file is part of Scapy
+ See http://www.secdev.org/projects/scapy for more informations
+ Copyright (C) Santiago Hernandez Ramos <shramos@protonmail.com>
+ This program is published under GPLv2 license '''
 
 
-from scapy.packet import *
-from scapy.fields import *
+from scapy.packet import Packet, bind_layers
+from scapy.fields import FieldLenField, BitEnumField, StrLenField, ShortField, \
+    ConditionalField, ByteEnumField, ByteField, StrNullField
 from scapy.layers.inet import TCP
+from scapy.error import Scapy_Exception
 
 
 # CUSTOM FIELDS
 
+# source: http://stackoverflow.com/questions/43711208/scapy-adding-a-new-field-with-a-dynamic-length?noredirect=1&lq=1
```

Comment:
Please use a **short** link to the **answer**: https://stackoverflow.com/a/43722441/3223422